CONTRIBUTING

```diff
@@ -14,6 +14,9 @@ To run tests
 pixi run test
 ```
 
+### Conventions
+We provide fixtures for example geff files that are exposed in conftest.py. If you use these fixtures, you can use `tests.utils.check_equiv_geff` to compare the geff generated by the fixture with the geff generated by your implementation.
```

Comment:
`conftest.py` doesn't actually exist at the moment, so we could mention it for future-proofing purposes or just refer to the `_pytest_plugin` file.

```suggestion
We provide fixtures for example geff files that are exposed in `src/geff/_pytest_plugin.py` or `tests/conftest.py`. If you use these fixtures, you can use `tests.utils.check_equiv_geff` to compare the geff generated by the fixture with the geff generated by your implementation.
```

Reply:
I suspect this needs to be further updated after today's decisions about the locations and exposing of tests/validation functions